lang/CMakeLists.txt

```
@@ -244,8 +242,7 @@ if(SC_HIDAPI)
   endif()
 endif()
 
-target_link_libraries(libsclang boost_thread_lib boost_system_lib boost_regex_lib boost_program_options_lib)
-target_include_directories(libsclang PUBLIC ${boost_include_dirs})
+target_link_libraries(libsclang Boost::thread Boost::program_options Boost::regex Boost::sync)
```

maybe make this multi-line so git history becomes more atomic?

```suggestion
target_link_libraries(
  libsclang Boost::thread
  Boost::program_options
  Boost::regex
  Boost::sync
)
```

Please only approve if the indentation is correct - I only eye-balled the indentation from the webui ;)

---

done (I've put `libsclang` and `Boost::thread` in separate lines though... ok?)